packages/marshal/src/assertPassStyleOf.js

```diff
@@ -0,0 +1,110 @@
+// @ts-check
+
+import { passStyleOf } from './passStyleOf.js';
+
+const { details: X, quote: q } = assert;
+
+/**
+ * @typedef {Array} CopyArray
+ */
+
+/**
+ * @typedef {Record<string, Passable>} CopyRecord
+ */
+
+/**
+ * Check whether the argument is a pass-by-copy array, AKA a "copyArray"
+ * in @agoric/marshal terms
+ *
+ * @param {CopyArray} array
+ * @returns {boolean}
+ */
+const isArray = array => passStyleOf(array) === 'copyArray';
```

Comment:
Be aware that the name `isArray` will often be confused with `Array.isArray`. No change necessarily suggested because I don't have one to suggest. But let me know if you think of something better.

Reply:
Hmm, we could change it to `isCArray` or `isCopyArray`

Reply:
I changed it to `isCopyArray`